libraries/ESP8266WiFi/src/ESP8266WiFiSTA.h

```diff
@@ -92,7 +93,13 @@ class ESP8266WiFiSTAClass {
 
     public:
 
+#ifdef NO_EXTRA_4K_HEAP

```

Comment:
Please add a descriptive comment about what the #define means, or at least reference the issue.